contrib/inventory/azure_rm.py

```diff
@@ -326,10 +326,10 @@ def __init__(self, args):
         # get authentication authority
         # for adfs, user could pass in authority or not.
         # for others, use default authority from cloud environment
-        if self.credentials.get('adfs_authority_url') is None:
-            self._adfs_authority_url = self._cloud_environment.endpoints.active_directory
-        else:
+        if self.credentials.get('adfs_authority_url'):
             self._adfs_authority_url = self.credentials.get('adfs_authority_url')
```

Comment:
i would do:

self._adfs_authority_url = self.credentials.get('adfs_authority_url') 
if self._adfs_authority_url is None:
   ...